src/main/java/nextstep/subway/domain/Distance.java

```diff
@@ -0,0 +1,34 @@
+package nextstep.subway.domain;
+
+import javax.persistence.Embeddable;
+import java.util.Objects;
+
+@Embeddable
+public class Distance {
+    private static final Long ZERO = 0L;
+    private final Long distance;
+
+    protected Distance() {
+        this(0L);
+    }
```

Comment:
👍 추가로 null이 허용되지 않기 때문에 primitive type을 활용해 볼 수 있을 것 같아요
```suggestion
   
    @Column(nullable = false)
    private int distance;

    protected Distance() {}
```

Reply:
네 step3 하기전에 반영하도록 하겠습니다 👍 
감사합니다^^